src/parseChangelog.ts

```diff
@@ -1,10 +1,18 @@
 import Changelog from './changelog';
-import { ChangeCategory, unreleased } from './constants';
+import {
+  ChangeCategory,
+  orderedChangeCategories,
+  unreleased,
+} from './constants';
 
 function truncated(line: string) {
   return line.length > 80 ? `${line.slice(0, 80)}...` : line;
 }
 
+function isValidChangeCategory(category: string): category is ChangeCategory {
+  return orderedChangeCategories.includes(category as ChangeCategory);
```

Comment:
I think we could just index into `ChangeCategory` since the values and keys are identical, but hardly worth blocking on.

Reply:
Great idea - done in https://github.com/MetaMask/auto-changelog/pull/62/commits/d903e1def68ee4bbb4a9af8d33e2c03920a71059